sql/updates/world/master/8888_88_88_88_world.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `spell_script_names` WHERE `spell_id`=167105 AND `ScriptName`='spell_warr_colossus_smash';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (167105, 'spell_warr_colossus_smash');
```

Comment:
Please put the content of this file into your already existing SQL file `9999_99_99_99_world.sql`. If you want to add more SQL commands for the script, use the same SQL file as long as it is used for the `world` database.